rootfs-builder/rootfs.sh

```diff
@@ -212,6 +212,7 @@ if [ -n "${USE_DOCKER}" ] ; then
 	# In case Clear Containers Runtime is installed we dont want to hit issue:
 	#https://github.com/clearcontainers/runtime/issues/828
 	docker run  \
+		--rm \
```

Comment:
Can't we also apply this to `image-builder/image_builder.sh`?

Reply:
yeah, that's right. same logic. update asap.😊